src/wasm/wasm-type.cpp

```diff
@@ -822,25 +815,24 @@ bool Type::isFunction() const {
 
 bool Type::isData() const {
   if (isBasic()) {
-    // The only basic type that is considered data is a string.
-    return isString();
+    return false;
```

Comment:
This change in classification seems nontrivial. Why is it ok?

Reply:
There are no basic types (note: not heap types) for which `isString` returns `true` 😆

Reply:
Ah yes, types and not heap types...